spock/plugins/helpers/inventory.py

```diff
@@ -34,27 +38,30 @@ def move_to_window(self, window, slot_nr):
         self.window, self.slot_nr = window, slot_nr
 
     def stacks_with(self, other):
-        if self.item_id != other.item_id: return False
-        if self.damage != other.damage: return False
-        if self.damage != other.damage: return False
-        # raise NotImplementedError('Stacks might differ by NBT data: %s %s' % (self, other))
-        # if self.nbt != other.nbt: return False  # TODO implement this correctly
+        if self.item_id != other.item_id:
+            return False
+        if self.damage != other.damage:
+            return False
+        # raise NotImplementedError('Stacks might differ by NBT data: %s %s'
+        # % (self, other))
+        # if self.nbt != other.nbt: return False
+        # TODO implement this correctly
         return self.max_amount != 1
 
     @property
     def max_amount(self):
         # TODO add the real values for ALL THE ITEMS! And blocks.
         # use some dummy values for now
-        if self.item_id in (-1,
-                256, 257, 258, 259, 261, 282,
-                326, 327, 333, 335, 342, 343, 346, 347, 355, 358, 359, 373, 374, 379, 380, 386, 387, 398,
-                403, 407, 408, 422, 417, 418, 419) \
-            or self.item_id in range(267, 280) \
-            or self.item_id in range(283, 287) \
-            or self.item_id in range(290, 295) \
-            or self.item_id in range(298, 318) \
-            or self.item_id in range(2256, 2268) \
-            : return 1
+        items_single = [-1, 256, 257, 258, 259, 261, 282, 326, 327, 333, 335,
+                        342, 343, 346, 347, 355, 358, 359, 373, 374, 379, 380,
+                        386, 387, 398, 403, 407, 408, 422, 417, 418, 419]
+        items_single.extend(range(267, 280))
+        items_single.extend(range(283, 287))
+        items_single.extend(range(290, 295))
+        items_single.extend(range(298, 318))
+        items_single.extend(range(2256, 2268))
+        for self.item_id in items_single:
```

Comment:
Will refactor this properly, but with that comment, I supposed it didn't really worth it to have a speed up. BTW, this is static, would convert it to a module level var (to just set it once)